desktop/src/app/app.component.ts

```diff
@@ -10,7 +10,7 @@ import { PermissionService } from "@batch-flask/ui/permission";
 import { EnvironmentMode, initEnvironment } from "@batch/ui-common";
 import { DependencyName } from "@batch/ui-common/lib/environment";
 import { DefaultFormControlResolver, DefaultFormLayoutProvider, FormControlResolver } from "@batch/ui-react/lib/components/form";
-import { StandardClock } from "@batch/ui-common/lib/datetime";
+import { StandardClock } from "@batch/ui-common/lib/datetime/standard-clock";
```

Comment:
Ran into some build issues with the original path so added "/standard-clock" which seemed to fix the build issues on my end.

Reply:
Fixed